recipes/libsecret/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "libsecret" %}
+{% set version = "0.18.5" %}
+{% set sha256 = "9ce7bd8dd5831f2786c935d82638ac428fa085057cc6780aba0e39375887ccb3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://ftp.gnome.org/pub/gnome/sources/libsecret/0.18/libsecret-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Disable-tests-which-require-additional-deps.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  # Uncomment when conda build >= 3
+  # run_exports:
+    #  https://abi-laboratory.pro/tracker/timeline/libsecret/
+    #  - libsecret >=0.18.5,<0.19
```

Comment:
Raising for pinning in PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/88 ). Please feel free to add more thoughts there.